prow/plugins/bugzilla/bugzilla.go

```diff
@@ -400,7 +415,11 @@ func validateBug(bug bugzilla.Bug, options plugins.BugzillaBranchOptions) (bool,
 	}
 
 	if options.Statuses != nil {
-		if !sets.NewString(*options.Statuses...).Has(bug.Status) {
+		validStatuses := sets.NewString(*options.Statuses...)
+		if options.StatusAfterValidation != nil {
+			validStatuses.Insert(*options.StatusAfterValidation)
```

Comment:
Would this perhaps be unexpected? (Is this too smart for itself?)

The alternative would be doing config validation to warn/fail if the `StatusAfterValidation` wasn't an explicitly allowed `validStatus`